languages/clojure/exercises/concept/basic-functions/test/basic-functions_test.clj

```diff
@@ -0,0 +1,13 @@
+(ns basic-functions-test
+  (:require [clojure.test :refer [deftest is]]
+            basic-functions))
+
+(deftest new-function-returns-42
+  (is (= 42 (new-function))))
```

Comment:
```suggestion
  (is (= 42 (basic-functions/new-function))))
```

Reply:
without namespace, lein test complains not found symbol new-function~

Reply:
that's odd. Since the file was required at the namespace level they all worked fine for me. 🤔